flaskbb/core/auth/registration.py

```diff
@@ -43,7 +43,7 @@ def validate(self, user_info):
 
         :param user_info: The provided registration information.
         :type user_info: :class:`~flaskbb.core.auth.registration.UserRegistrationInfo`
-        """
+        """  # noqa
```

Comment:
lol I didn't even know this was possible but it makes Flake8 happy without doing something dumb here.